src/Engine/RadiumEngine.cpp

```diff
@@ -206,7 +215,8 @@ bool RadiumEngine::loadFile( const std::string& filename ) {
     return true;
 }
 
-void RadiumEngine::releaseFile() {
+void RadiumEngine::releaseFile(const std::string &filename)
```

Comment:
actually this is WIP right ?

Reply:
This is a first step toward more general functionality. For the moment, it just clarify the API. Both `loadFile` and `releaseFile` take the filename.